cmd/meroxa/root/apps/logs_test.go

```diff
@@ -93,7 +107,7 @@ func TestApplicationLogsExecution(t *testing.T) {
 	}
 
 	gotLeveledOutput := logger.LeveledOutput()
-	wantLeveledOutput := display.AppLogsTable(appLogs)
```

Comment:
I think `AppLogsTable` could be removed entirely (with its test), and maybe leave the `AppLogsTableV2` as `AppLogsTable`.

Reply:
addressed in https://github.com/meroxa/cli/pull/804/commits/98953fabe1801055e95b650c0f438d40b860e798